packages/liveblocks-react/src/types.ts

```diff
@@ -31,6 +31,16 @@ import type {
   ToImmutable,
 } from "@liveblocks/core";
 
+export type UseStorageStatusOptions = {
+  /**
+   * When setting smooth, the hook will not update immediately as status
+   * changes. This is because in typical applications, these states can change
+   * quickly between synchronizing and synchronized. If you use this hook to
+   * build a "Saving changes..." style UI, prefer setting `smooth: true`.
+   */
+  smooth?: 0 | 1 | 2;
```

Comment:
To be clear: this is not the final API. Just adding these here for now to test two alternative possible implementatoins.